matminer/featurizers/site.py

```diff
@@ -464,14 +472,31 @@ def featurize(self, struct, idx):
 
         cn_fingerprint = []
 
+        if self.chem_info is not None:
+            absdeltas = {}
```

Comment:
suggest refactor this variable to be named:

``prop_absdelta`` since it is a dictionary of properties to delta values